3-Lines/Lesson-15.js

```diff
@@ -0,0 +1,22 @@
+// Lesson-5 Quotation marks
+
+// 'Hello'
+// 'Goodbye'
+// 'G'
+// ' '
+// ''
+// Любой одиночный символ в кавычках — это строка. Пустая строка '' — это тоже строка.
+
+// Стандарт кодирования airbnb, рекомендует использовать, по возможности, одинарные кавычки.
+console.log('Dracarys!');
+
+// Если мы хотим написать кавычки, то необходимо воспользоваться экраном - \
+// Экранируется только ", так как в этой ситуации двойные кавычки имеют специальное значение
+console.log("Dragon's mother said \"No\""); 
```

Comment:
FYI: Modern linter configs prefer ' over " and will complain about string literals enclosed with "". 
FYI2: If we use different quotes, there is no need in backslash
```javascript
console.log('She said "No"');
console.log("She said 'No'");
```

Reply:
I agree with the comments. 
In the specified example, console.log("Dragon's mother said \"No\"") also uses linter ', which somehow leads to escaping either 'or', which is the purpose of the task.